lib/reportAProblem/processData.js

```diff
@@ -0,0 +1,26 @@
+export const processData = (data) => {
+  // loop over keys in data and generate new object that aligns to notify body
+  return Object.keys(data).reduce((prevVal, currentVal) => {
+    // split value according to capitalized letters, make all lower case, and then join by underscore
+    let variableName = currentVal
+      .split(/(?=[A-Z])/)
+      .map((value) => value.toLowerCase())
+      .join("_");
```

Comment:
Is this basically a translation between Javascript v. Python conventions? Would it be easier to use the `snake_case` variable names in the HTML and then never convert them?

Reply:
that's probably a better idea yeah